tracer/test/snapshots/Iast.HeaderInjection.AspNetCore5.Vuln.Cookie.SensitiveValue.verified.txt

```diff
@@ -73,9 +71,7 @@
       component: aspnet_core,
       env: integration_tests,
       language: dotnet,
-      span.kind: server,
```

Comment:
Changing the debug properties of the project makes this information disappear from the snapshots? I mean, it's fine if it's not shown, I'm just curious

Reply:
The `span.kind: server` isn't removed here, only the comma at the end is removed because it's now the last tag, because the last 2 tags are removed. The diff is broken.

If you were talking about the `_dd.git.commit.sha` and `_dd.git.repository_url`, these both tags are generated with the needs of symbols. But in our case we stop the generation to get a consistent data between all frameworks, archs and plateforms (cf my other comment). That's why the doesn't get produced anymore 😄